custom/icds_reports/utils/__init__.py

```diff
@@ -587,7 +588,12 @@ def generate_data_for_map(data, loc_level, num_prop, denom_prop, fill_key_lower,
     total = 0
     values_to_calculate_average = {'numerator': 0, 'denominator': 0}
 
+
     for row in data:
+        if location_launched_status is not None:
```

Comment:
do you have a screenshot of what it looks like when this kind of filtering is done